error.h

```diff
@@ -3,10 +3,14 @@
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <stdlib.h>
 #include "message_types.h"
 
 // how long the char array you pass to get_next_serialized_error needs to be
-#define ERROR_COMMAND_LENGTH 5
+#define ERROR_COMMAND_LENGTH 8
+
+// How many error messages can be buffered at once
+#define ERROR_MESSAGE_RING_BUFFER_SIZE 16
 
 typedef struct {
```

Comment:
I don't think any other code modules are going to use this type, it can probably be moved to the c file instead of the h file